src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1502,6 +1502,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return NewConverterInstance(assemblyQualifiedtypeName);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Drawing.Primitives.dll; SideIndicator=<=}.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 126
126:         $diffs | Should -BeExactly $null
```